backend/src/main/java/pl/edu/agh/websocket/RefereeDecision.java

```diff
@@ -14,6 +14,21 @@
 public class RefereeDecision {
     private Integer fighter1Points;
     private Integer fighter2Points;
-    private List<String> decision;
+    private List<Event> decision;
```

Comment:
needs discussion

Reply:
Does this Event class have any influence on the code on the backend side/app logic? Or is it related to a database? ActionInfo field will keep the same data as the present String? On the front, it is requiring now changes in some places to fit the decision to an Event Object and also map it when requesting its content. I would prefer to leave it as it is, but if it forces changes on you, it would be great to have a call and see where the editing is less invasive.